test/ex_image_info_test/images/bmp_test.exs

```diff
@@ -14,7 +14,7 @@ defmodule ExImageInfoTest.Images.BMPTest do
   test "force - bmp disk image - #seems? #type #info", images do
     assert seems?(images["bmp"], :bmp) == true
     assert type(images["bmp"], :bmp) == {"image/bmp", "BMP"}
-    assert info(images["bmp"], :bmp) == {"image/bmp",130, 42, "BMP"}
+    assert info(images["bmp"], :bmp) == {"image/bmp", 130, 42, "BMP"}
   end
 
   # @tag :wip
```

Comment:
Maybe include the removal of these unneeded comments?
```suggestion

```

Reply:
Yea we can knock those out too

Reply:
Done.